docs/tutorials/aselite.md

```diff
@@ -0,0 +1,23 @@
+# How to use Aselite flow
+
+Aselite is a prefect flow that extract data from SQL Server database and save collected data as .csv in Azure Data Lake in defined folder.
```

Comment:
1. Same as above, the name of the flow is `ASELiteToADLS`
2. While we're at it, I think Prefect should be capitalized (unless you're referring to the library, then you should use backticks (`prefect`).